vmm/src/device_manager.rs

```diff
@@ -4119,6 +4119,29 @@ fn numa_node_id_from_memory_zone_id(numa_nodes: &NumaNodes, memory_zone_id: &str
     None
 }
 
+struct TPMDevice {}
```

Comment:
Per Rust expectation about structures' names:
```suggestion
struct TpmDevice {}
```